src/metabase/api/dashboard.clj

```diff
@@ -379,49 +379,6 @@
     (validation/check-embedding-enabled)
     (api/check-superuser)))
 
-(api/defendpoint PUT "/:id"
```

Comment:
This was moved by a PR on master to elsewhere in the file but got re-added during the rebase, so I'm removing it again here.